tests/sources/eltwise_binary_test.cc

```diff
@@ -0,0 +1,88 @@
+#include <cstdint>
+#include <cstdio>
+
+#include "llk_defs.h"
+#include "ckernel.h"
+
+// Globals
+uint32_t unp_cfg_context = 0;
+uint32_t pack_sync_tile_dst_ptr = 0;
+volatile uint32_t tt_l1_ptr l1_buffer[16] __attribute__ ((section (".text#"))) __attribute__ ((aligned (16)));
```

Comment:
```suggestion
volatile std::uint32_t tt_l1_ptr l1_buffer[16] __attribute__ ((section (".text#"), aligned (16)));
```
But more importantly, why do we want this array in the .text section? It sounds a bit odd to be honest. Could you elaborate on that? Even better, including an explanation in form of a comment would be nice.